monicar-collector/src/main/java/org/collector/consumer/CycleInfoConsumer.java

```diff
@@ -17,6 +22,9 @@ public class CycleInfoConsumer {
 	@KafkaListener(topicPartitions = @TopicPartition(topic = "cycleInfo-json-topic",
 		partitions = "#{@finder.partitions('cycleInfo-json-topic')}"))
 	public void accept(ConsumerRecord<String, CycleInfoRequest> message) {
+		if(message.value().cList().size() != CycleInfoSize.CYCLE_INFO_MAX_SIZE.getSize()){
+			throw new CustomException(ResponseCode.NOT_CYCLE_INFO_SIZE_ERROR);
+		}
```

Comment:
CycleInfoSize enum 내에서 검증하면 더 깔끔해 보일 것 같아요
```suggestion
		if(CycleInfoSize.MAX.validate(message.value().cList().size())) {}
```

```
// CycleInfoSize.java
    public void validate(int listSize) {
        if (listSize != this.size) {
            throw new CustomException(ResponseCode.NOT_CYCLE_INFO_SIZE_ERROR);
        }
    }
```

Reply:
7d394c4061e606f4aa1137b69afa4c16c034725b 에서 확인가능합니다.